### PR TITLE
8343474: [updates] Customize README.md to specifics of update project

### PR DESCRIPTION
**Pull request by GoeLin**
Customize the README.md to jdk21u.

Plan is to backport this to 17 and maybe older jdks and adapt the paths etc correspondingly.
So let’s keep this sufficiently generic so that the information holds for jdkXu and jdkXu-dev
and that the change can easily be backported (by adapting 21->17 etc).

Should we put there more information?

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] [JDK-8343474](https://bugs.openjdk.org/browse/JDK-8343474) needs maintainer approval
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8343474](https://bugs.openjdk.org/browse/JDK-8343474): [updates] Customize README.md to specifics of update project (**Bug** - P4 - Approved)

### Reviewers
 * [Christoph Langer](https://openjdk.org/census#clanger) (@RealCLanger - **Reviewer**)
 * [Severin Gehwolf](https://openjdk.org/census#sgehwolf) (@jerboaa - **Reviewer**)
 * [Andrew John Hughes](https://openjdk.org/census#andrew) (@gnu-andrew - **Reviewer**)
 * [Andrew Haley](https://openjdk.org/census#aph) (@theRealAph - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk21u-dev.git pull/1118/head:pull/1118` \
`$ git checkout pull/1118`

Update a local copy of the PR: \
`$ git checkout pull/1118` \
`$ git pull https://git.openjdk.org/jdk21u-dev.git pull/1118/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1118`

View PR using the GUI difftool: \
`$ git pr show -t 1118`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk21u-dev/pull/1118.diff">https://git.openjdk.org/jdk21u-dev/pull/1118.diff</a>

</details>
<details><summary>Using Webrev</summary>

[Link to Webrev Comment](https://git.openjdk.org/jdk21u-dev/pull/1118#issuecomment-2452975654)
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
-# Welcome to the JDK!
+# Welcome to OpenJDK 21 Updates!
+
+JDK 21 updates are continuously developed in repository [jdk21u-dev](https://github.com/openjdk/jdk21u-dev).
+Repository [jdk21u](https://github.com/openjdk/jdk21u) is used for rampdown of the update releases of jdk21u and only accepts critical changes that must make the next release during rampdown. (You probably do not want to target jdk21u).
+
+For more OpenJDK 21 updates specific information as timelines and contribution see the [project wiki page](https://wiki.openjdk.org/display/JDKUpdates/JDK+21u/).
```
Comment (RealCLanger) [inline README.md:6]: Maybe 
```suggestion
For more OpenJDK 21 updates specific information such as timelines and contribution guidelines see the [project wiki page](https://wiki.openjdk.org/display/JDKUpdates/JDK+21u/).
```

**Review by RealCLanger — APPROVED** [review]

**Review by jerboaa — APPROVED** [review]
Ship it!

File: README.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# Welcome to the JDK!
+# Welcome to OpenJDK 21 Updates!
+
+JDK 21 updates are continuously developed in repository [jdk21u-dev](https://github.com/openjdk/jdk21u-dev).
+Repository [jdk21u](https://github.com/openjdk/jdk21u) is used for rampdown of the update releases of jdk21u and only accepts critical changes that must make the next release during rampdown. (You probably do not want to target jdk21u).
```
Comment (theRealAph) [inline README.md:4]:
```suggestion
The JDK 21 Updates project uses two GitHub repositories.
Updates are continuously developed in the repository [jdk21u-dev](https://github.com/openjdk/jdk21u-dev). This is the repository usually targeted by contributors.
The [jdk21u](https://github.com/openjdk/jdk21u) repository is used for rampdown of the update releases of jdk21u and only accepts critical changes that must make the next release during rampdown. (You probably do not want to target jdk21u).
```

**Review by gnu-andrew — COMMENTED** [review]
I would go with aph's version, which reads a little better.

**Review by gnu-andrew — APPROVED** [review]

**Review by theRealAph — APPROVED** [review]

**Review by RealCLanger — APPROVED** [review]

**Comment by GoeLin** [comment]
Thanks for the reviews and help with this!

**Pull request closed without merging**